package.json

```diff
@@ -38,9 +38,10 @@
     "test-api": "mocha -R dot --recursive tests/api",
     "test-disablehttp": "DISABLE_HTTP=true mocha -R dot --recursive tests/disableHttp",
     "test-enforced": "USE_ACCESS_TOKEN=true mocha -R dot --recursive tests/enforceToken",
+    "test-cache-persp": "ENABLE_CACHE_PERSPECTIVE=true mocha -R dot --recursive tests/enableCache",
```

Comment:
Oh, I see.

OK...

Since the perspectives controller uses the api/v1/helpers/nouns/perspectives.js helper and passes it around to the various utility functions, what about setting the value of the helper's cacheEnabled attribute in each of the perspective controller functions which needs it, e.g.

```
  getPerspective(req, res, next) {
    helper.cacheEnabled = featureToggles.isFeatureEnabled('enableCachePerspective');
    doGet(req, res, next, helper);
  },
```